invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsModal.js

```diff
@@ -235,108 +234,14 @@ export class CreatibutorsModal extends Component {
     }
   };
 
-  makeIdEntry = (identifier) => {
-    let icon = null;
-    let link = null;
-
-    if (identifier.scheme === "orcid") {
-      icon = "/static/images/orcid.svg";
-      link = "https://orcid.org/" + identifier.identifier;
-    } else if (identifier.scheme === "gnd") {
-      icon = "/static/images/gnd-icon.svg";
-      link = "https://d-nb.info/gnd/" + identifier.identifier;
-    } else if (identifier.scheme === "ror") {
-      icon = "/static/images/ror-icon.svg";
-      link = "https://ror.org/" + identifier.identifier;
-    } else if (identifier.scheme === "isni" || identifier.scheme === "grid") {
-      return null;
-    } else {
-      return (
-        <>
-          {identifier.scheme}: {identifier.identifier}
-        </>
-      );
-    }
-
-    return (
-      <span key={identifier.identifier}>
-        <a href={link} target="_blank" rel="noopener noreferrer">
-          <Image
-            src={icon}
-            className="inline-id-icon ml-5 mr-5"
-            verticalAlign="middle"
-          />
-          {identifier.scheme === "orcid" ? identifier.identifier : null}
-        </a>
-      </span>
-    );
-  };
-
-  serializeSuggestions = (creatibutors) => {
-    const results = creatibutors.map((creatibutor) => {
-      // ensure `affiliations` and `identifiers` are present
-      creatibutor.affiliations = creatibutor.affiliations || [];
-      creatibutor.identifiers = creatibutor.identifiers || [];
-
-      let affNames = "";
-      if ("affiliations" in creatibutor) {
-        creatibutor.affiliations.forEach((affiliation, idx) => {
-          affNames += affiliation.name;
-          if (idx < creatibutor.affiliations.length - 1) {
-            affNames += ", ";
-          }
-        });
-      }
-
-      const idString = [];
-      creatibutor.identifiers?.forEach((i) => {
-        idString.push(this.makeIdEntry(i));
-      });
-      const { isOrganization } = this.state;
-
-      return {
-        text: creatibutor.name,
-        value: creatibutor.id,
-        extra: creatibutor,
-        key: creatibutor.id,
-        content: (
-          <Header>
-            {creatibutor.name} {idString.length ? <>({idString})</> : null}
-            <Header.Subheader>
-              {isOrganization ? creatibutor.acronym : affNames}
-            </Header.Subheader>
-          </Header>
-        ),
-      };
-    });
-
-    const { showPersonForm } = this.state;
+  SuggestionsWrapper = (creatibutors) => {
```

Comment:
we typically use this name convention for classes. Shoulnd't that be just a serialize method that returns the `AffiliationsSuggestions`?

Reply:
yep! the naming is still WIP will change